manifests/overlays/standalone/kustomization.yaml

```diff
@@ -2,12 +2,13 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: kubeflow
 resources:
-  - ../../base
```

Comment:
I would suggest to revert the formatting changes. If they are done automatically we can have a dedicated PR for them

Reply:
Reverted, thanks for pointing out